Update/onik_004.txt

```diff
@@ -2590,7 +2600,7 @@ void main()
 //　一通り納得すると、彼もまたレナの姿を求め駆け出して行った＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　一通り納得すると、彼もまたレナの姿を求め駆け出して行った。",
-		   NULL, "Satisfied with that, he also rushed off to look for Rena.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "Satisfied with that, she also rushed off to look for Rena.", GetGlobalFlag(GLinemodeSp));
```

Comment:
I think the pronoun is Keiichi in this case, so "he" is correct.

Reply:
This seems to be keiichi's inner monologue and here he refers to the girl who talked above. So "she" is correct here in my opinion.